.github/workflows/playwright.yml

```
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
```

but in such cases -- only 2
```suggestion
      - uses: actions/checkout@v4
```

---

fixed

---

fixed incorrectly